modules/rhel-compute-updating.adoc

```diff
@@ -8,6 +8,13 @@
 After you update your cluster, you must update the Red Hat Enterprise Linux (RHEL)
 compute machines in your cluster.
 
+[IMPORTANT]
+====
+You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. You must deploy new {op-system-base} 8 hosts, and the old {op-system-base} 7 hosts should be removed.
+====
+
+// TODO: This module needs to be updated to reflect RHEL 8 compute machines in 4.10. Because initial support for RHEL 8 starts in 4.9, and upgrading RHEL 7 -> 8 in-place is not supported, this is being left to reflect RHEL 7 upgrades.
+
 .Prerequisites
 
 * You updated your cluster.
```

Comment:
Shall we remove the following content as it's for upgrading RHEL-7 workers?

Reply:
hold on...we should be able to upgrade the OCP-4.8 with RHEL-7 workers to OCP-4.9 with RHEL-7.9 worker.
So this is still useful for users.
But I noticed there's something we need to update:
```
# subscription-manager repos --disable=rhel-7-server-ose-4.7-rpms \
                             --enable=rhel-7-server-ansible-2.9-rpms \
                             --enable=rhel-7-server-ose-4.8-rpms
```

```
# subscription-manager repos --disable=rhel-7-server-ose-4.7-rpms \
                             --enable=rhel-7-server-ose-4.8-rpms  \
                             --enable=rhel-7-fast-datapath-rpms   \
                             --enable=rhel-7-server-optional-rpms
```
we may need to update the `4.7` to `4.8` and `4.8` to `4.9` in above commands.

Reply:
I have pushed new updates for those versions :+1: